epsagon/modules/logging.py

```diff
@@ -24,10 +28,41 @@ def _wrapper(wrapped, _instance, args, kwargs):
     return wrapped(*args, **kwargs)
 
 
+def _get_log_id():
+    """
+    retrieve or generate the log id for this trace
+    """
+    return trace_factory.get_log_id()
```

Comment:
let's not generate this log id for Lambda and StepLambda runners, since we don't need them there
I'm not sure where to do it in the instrumentation actually - let's think about it

Reply:
It will only generate it if the flag is set to true, which will not happen for lambdas..